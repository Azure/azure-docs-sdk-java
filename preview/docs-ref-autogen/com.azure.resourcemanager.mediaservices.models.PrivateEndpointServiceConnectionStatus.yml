### YamlMime:JavaType
uid: "com.azure.resourcemanager.mediaservices.models.PrivateEndpointServiceConnectionStatus"
fullName: "com.azure.resourcemanager.mediaservices.models.PrivateEndpointServiceConnectionStatus"
name: "PrivateEndpointServiceConnectionStatus"
nameWithType: "PrivateEndpointServiceConnectionStatus"
summary: "The private endpoint connection status."
inheritances:
- "<xref href=\"java.lang.Object?displayProperty=fullName\" data-throw-if-not-resolved=\"False\" />"
- "<xref href=\"com.azure.core.util.ExpandableStringEnum?displayProperty=fullName\" data-throw-if-not-resolved=\"False\" />"
inheritedMembers:
- "com.azure.core.util.ExpandableStringEnum.<T>fromString(java.lang.String,java.lang.Class<T>)"
- "com.azure.core.util.ExpandableStringEnum.<T>values(java.lang.Class<T>)"
- "com.azure.core.util.ExpandableStringEnum.equals(java.lang.Object)"
- "com.azure.core.util.ExpandableStringEnum.hashCode()"
- "com.azure.core.util.ExpandableStringEnum.toString()"
- "java.lang.Object.clone()"
- "java.lang.Object.finalize()"
- "java.lang.Object.getClass()"
- "java.lang.Object.notify()"
- "java.lang.Object.notifyAll()"
- "java.lang.Object.wait()"
- "java.lang.Object.wait(long)"
- "java.lang.Object.wait(long,int)"
syntax: "public final class PrivateEndpointServiceConnectionStatus extends ExpandableStringEnum<PrivateEndpointServiceConnectionStatus>"
constructors:
- uid: "com.azure.resourcemanager.mediaservices.models.PrivateEndpointServiceConnectionStatus.PrivateEndpointServiceConnectionStatus()"
  fullName: "com.azure.resourcemanager.mediaservices.models.PrivateEndpointServiceConnectionStatus.PrivateEndpointServiceConnectionStatus()"
  name: "PrivateEndpointServiceConnectionStatus()"
  nameWithType: "PrivateEndpointServiceConnectionStatus.PrivateEndpointServiceConnectionStatus()"
  syntax: "public PrivateEndpointServiceConnectionStatus()"
fields:
- uid: "com.azure.resourcemanager.mediaservices.models.PrivateEndpointServiceConnectionStatus.APPROVED"
  fullName: "com.azure.resourcemanager.mediaservices.models.PrivateEndpointServiceConnectionStatus.APPROVED"
  name: "APPROVED"
  nameWithType: "PrivateEndpointServiceConnectionStatus.APPROVED"
  summary: "Static value Approved for Private<wbr>Endpoint<wbr>Service<wbr>Connection<wbr>Status."
  modifiers:
  - "static"
  - "final"
  field:
    type: "<xref href=\"com.azure.resourcemanager.mediaservices.models.PrivateEndpointServiceConnectionStatus?alt=com.azure.resourcemanager.mediaservices.models.PrivateEndpointServiceConnectionStatus&text=PrivateEndpointServiceConnectionStatus\" data-throw-if-not-resolved=\"False\" />"
  syntax: "public static final PrivateEndpointServiceConnectionStatus APPROVED"
  desc: "Static value Approved for PrivateEndpointServiceConnectionStatus."
- uid: "com.azure.resourcemanager.mediaservices.models.PrivateEndpointServiceConnectionStatus.PENDING"
  fullName: "com.azure.resourcemanager.mediaservices.models.PrivateEndpointServiceConnectionStatus.PENDING"
  name: "PENDING"
  nameWithType: "PrivateEndpointServiceConnectionStatus.PENDING"
  summary: "Static value Pending for Private<wbr>Endpoint<wbr>Service<wbr>Connection<wbr>Status."
  modifiers:
  - "static"
  - "final"
  field:
    type: "<xref href=\"com.azure.resourcemanager.mediaservices.models.PrivateEndpointServiceConnectionStatus?alt=com.azure.resourcemanager.mediaservices.models.PrivateEndpointServiceConnectionStatus&text=PrivateEndpointServiceConnectionStatus\" data-throw-if-not-resolved=\"False\" />"
  syntax: "public static final PrivateEndpointServiceConnectionStatus PENDING"
  desc: "Static value Pending for PrivateEndpointServiceConnectionStatus."
- uid: "com.azure.resourcemanager.mediaservices.models.PrivateEndpointServiceConnectionStatus.REJECTED"
  fullName: "com.azure.resourcemanager.mediaservices.models.PrivateEndpointServiceConnectionStatus.REJECTED"
  name: "REJECTED"
  nameWithType: "PrivateEndpointServiceConnectionStatus.REJECTED"
  summary: "Static value Rejected for Private<wbr>Endpoint<wbr>Service<wbr>Connection<wbr>Status."
  modifiers:
  - "static"
  - "final"
  field:
    type: "<xref href=\"com.azure.resourcemanager.mediaservices.models.PrivateEndpointServiceConnectionStatus?alt=com.azure.resourcemanager.mediaservices.models.PrivateEndpointServiceConnectionStatus&text=PrivateEndpointServiceConnectionStatus\" data-throw-if-not-resolved=\"False\" />"
  syntax: "public static final PrivateEndpointServiceConnectionStatus REJECTED"
  desc: "Static value Rejected for PrivateEndpointServiceConnectionStatus."
methods:
- uid: "com.azure.resourcemanager.mediaservices.models.PrivateEndpointServiceConnectionStatus.fromString(java.lang.String)"
  fullName: "com.azure.resourcemanager.mediaservices.models.PrivateEndpointServiceConnectionStatus.fromString(String name)"
  name: "fromString(String name)"
  nameWithType: "PrivateEndpointServiceConnectionStatus.fromString(String name)"
  summary: "Creates or finds a Private<wbr>Endpoint<wbr>Service<wbr>Connection<wbr>Status from its string representation."
  modifiers:
  - "static"
  parameters:
  - description: "a name to look for."
    name: "name"
    type: "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/String.html\">String</a>"
  syntax: "public static PrivateEndpointServiceConnectionStatus fromString(String name)"
  desc: "Creates or finds a PrivateEndpointServiceConnectionStatus from its string representation."
  returns:
    description: "the corresponding PrivateEndpointServiceConnectionStatus."
    type: "<xref href=\"com.azure.resourcemanager.mediaservices.models.PrivateEndpointServiceConnectionStatus?alt=com.azure.resourcemanager.mediaservices.models.PrivateEndpointServiceConnectionStatus&text=PrivateEndpointServiceConnectionStatus\" data-throw-if-not-resolved=\"False\" />"
- uid: "com.azure.resourcemanager.mediaservices.models.PrivateEndpointServiceConnectionStatus.values()"
  fullName: "com.azure.resourcemanager.mediaservices.models.PrivateEndpointServiceConnectionStatus.values()"
  name: "values()"
  nameWithType: "PrivateEndpointServiceConnectionStatus.values()"
  summary: "Gets known Private<wbr>Endpoint<wbr>Service<wbr>Connection<wbr>Status values."
  modifiers:
  - "static"
  syntax: "public static Collection<PrivateEndpointServiceConnectionStatus> values()"
  desc: "Gets known PrivateEndpointServiceConnectionStatus values."
  returns:
    description: "known PrivateEndpointServiceConnectionStatus values."
    type: "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/util/Collection.html\">Collection</a>&lt;<xref href=\"com.azure.resourcemanager.mediaservices.models.PrivateEndpointServiceConnectionStatus?alt=com.azure.resourcemanager.mediaservices.models.PrivateEndpointServiceConnectionStatus&text=PrivateEndpointServiceConnectionStatus\" data-throw-if-not-resolved=\"False\" />&gt;"
type: "class"
desc: "The private endpoint connection status."
metadata: {}
package: "com.azure.resourcemanager.mediaservices.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-mediaservices:2.2.0-beta.1
