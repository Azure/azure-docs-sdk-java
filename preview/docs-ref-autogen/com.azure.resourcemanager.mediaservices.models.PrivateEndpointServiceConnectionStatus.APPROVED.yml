### YamlMime:JavaMember
uid: "com.azure.resourcemanager.mediaservices.models.PrivateEndpointServiceConnectionStatus.APPROVED*"
fullName: "com.azure.resourcemanager.mediaservices.models.PrivateEndpointServiceConnectionStatus.APPROVED"
name: "APPROVED"
nameWithType: "PrivateEndpointServiceConnectionStatus.APPROVED"
members:
- uid: "com.azure.resourcemanager.mediaservices.models.PrivateEndpointServiceConnectionStatus.APPROVED"
  fullName: "com.azure.resourcemanager.mediaservices.models.PrivateEndpointServiceConnectionStatus.APPROVED"
  name: "APPROVED"
  nameWithType: "PrivateEndpointServiceConnectionStatus.APPROVED"
  summary: "Static value Approved for PrivateEndpointServiceConnectionStatus."
  syntax: "public static final PrivateEndpointServiceConnectionStatus APPROVED"
type: "field"
metadata: {}
package: "com.azure.resourcemanager.mediaservices.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-mediaservices:1.0.0-beta.1
